bitrise.yml

```diff
@@ -59,6 +59,12 @@ workflows:
     - errcheck:
     - go-test:
 
+  maintenance:
+    steps:
+    - go-test:
+        inputs:
+          - packages: ./maintenance
```

Comment:
did you forget to push the maintenance package?

Reply:
indeed